cms/management/commands/build_frontend.py

```
@@ -0,0 +1,124 @@
+            self.stdout.write(f'Installing dependencies for {package_name}...')
+            self.run_npm_command(package_path, 'install', verbose)
+
+        # Run build
+        self.run_npm_command(package_path, 'run build', verbose)
+        self.stdout.write(self.style.SUCCESS(f'✓ {package_name} built successfully'))
+        return True
+
+    def run_npm_command(self, working_dir, command, verbose=False):
+        cmd = f'npm {command}'
+        try:
+            result = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call with `shell=True` identified, security issue

(S602)